### PR TITLE
Fixing dependencies listed in requirements.txt file

### PR DESCRIPTION
**Pull request by ortengren**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+pluggy>=1.5.0
+pyparsing>=3.1.0
+pytest>=8.2.0
+python-dateutil>=2.9.0.post0
+
+rascaline @ https://luthaf.fr/nightly-wheels/rascaline/rascaline-0.1.0.dev558-py3-none-manylinux_2_17_x86_64.manylinux2014_x86_64.whl; sys_platform=="linux"
+
+rascaline @ https://luthaf.fr/nightly-wheels/rascaline/rascaline-0.1.0.dev558-py3-none-macosx_11_0_arm64.whl; sys_platform=="darwin" and platform_machine=="arm64"
+
+rascaline @ https://luthaf.fr/nightly-wheels/rascaline/rascaline-0.1.0.dev558-py3-none-macosx_10_9_x86_64.whl; sys_platform=="darwin" and platform_machine=="x86_64"
```
Comment (rosecers) [inline requirements.txt:33]: I would probably suggest not pinning this to a specific platform or version -- does it not build from the github anymore?

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+fonttools>=4.53.0
+importlib_resources>=6.4.0
+iniconfig>=2.0.0
+kiwisolver>=1.4.5
+matplotlib>=3.9.0
+
+metatensor==0.2.0
+metatensor-core==0.1.8
+metatensor-learn==0.2.2
+metatensor-operations==0.2.2
```
Comment (rosecers) [inline requirements.txt:19]: Do we want to pin these to specific versions? It is usually good to use >= whenever we can

File: requirements.txt (hunk 1 above)
Comment (ortengren) [reply requirements.txt:33]: I was running into issues with rascaline, I think due to the compilation process, so I set it to install from the pre-built wheels instead.  I will double check this, though.

File: requirements.txt (hunk 2 above)
Comment (ortengren) [reply requirements.txt:19]: I can update these, I was just worried about stability issues from possible updates.

**Comment by arthur-lin1027** [comment]
Lucas, I simplifed the requirements.txt. Can you go into one of your isolated systems (one with miniconda and a rust compiler), and run 
`pip install -r requirements.txt` and `pip install .` to see if anisoap properly installs?

**Comment by arthur-lin1027** [comment]
I'm going to close this PR because it's confusing. 
Let's try to treat documentation updates like feature branches which get deleted after merged into main.

**Pull request closed without merging**